queries/aws_queries/AWS_Authentication_from_CrowdStrike_Unmanaged_Device_FDREvent.yml

```diff
@@ -0,0 +1,19 @@
+AnalysisType: scheduled_query
```

Comment:
Would it be easy to add a comment both on the original query YAML file and vice versa, since they'll have to be maintained in parallel for a short period?

Reply:
Something like the comment here: https://github.com/panther-labs/panther-analysis/pull/648/commits/e867affb2c3f6b7e9fdc6a7cc94c88fad2e529fd

Reply:
Nice, but perhaps consider adding the entire Github permalink?

Reply:
Sure. Added.